languages/ocaml.scm

```diff
@@ -87,7 +100,46 @@
   "<-"
   "{"
   "}"
-] @prepend_space @append_space
+] @append_space
+
+(
+  "("* @do_nothing
+  .
+  [
+    "as"
+    "assert"
+    "begin"
+    "do"
+    "else"
+    "exception"
+    "external"
+    (infix_operator)
+    "let"
+    "match"
+    "module"
+    "mutable"
+    "of"
+    "open"
+    (parameter)
+    "rec"
+    "sig"
+    "then"
+    "try"
+    "type"
+    "val"
+    "when"
+    "while"
+    "with"
+    "*"
+    "="
+    "|"
+    "||"
+    "->"
+    "<-"
+    "{"
+    "}"
+  ] @prepend_space
+)
 
 ; Prepend spaces
```

Comment:
I think you can merge all of these with the above that checks for an opening paren.